cli/utilities/packages.py

```diff
@@ -62,11 +62,55 @@ def install(self, pkg, nogpgcheck=False):
         Args:
             pkg (str): package need to be installed
         """
-        cmd = f"{self.manager} install -y {pkg}"
-        if nogpgcheck:
-            cmd += " --nogpgcheck"
-        if self.execute(sudo=True, long_running=True, cmd=cmd):
-            raise PackageError(f"Failed to install package '{pkg}'")
+        # If the package to be installed is docker, follow
+        # the steps in https://docs.docker.com/engine/install/rhel/
+        if pkg == "docker":
```

Comment:
These steps are not recommended. Installation guide doesn't mention any such process to install docker.